lib/charms/opensearch/v0/opensearch_users.py

```diff
@@ -47,16 +47,15 @@ def create_role(
         Raises:
             OpenSearchUserMgmtError: If the role creation request fails.
         """
-        put_role_resp = self.opensearch.request(
+        resp = self.opensearch.request(
             "PUT",
             f"{ROLE_ENDPOINT}/{role_name}",
             {**(permissions or {}), **(action_groups or {})},
         )
-        logger.debug(put_role_resp)
-        if put_role_resp.get("status") != "OK":
-            raise OpenSearchUserMgmtError(
-                f"creating role {role_name} failed - response: {put_role_resp}"
-            )
+        logger.debug(resp)
```

Comment:
is a password or certificate exposed here?

Reply:
Not by default; passwords and hashes are [hidden resources](https://opensearch.org/docs/latest/security/access-control/api/#reserved-and-hidden-resources), so they're removed from API output, and I can't find a documented option to make them visible. However, I don't think these lines are necessary, so I'm going to remove them anyway.